### PR TITLE
Blog post: Cake support in renovate

### PR DESCRIPTION
**Pull request by nils-a**

### CODE REVIEW

**Review by augustoproiete — APPROVED** [review]
LGTM!

File: input/blog/2021-04-13-cake-support-in-renovate.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### Enabling Renovate (on GitHub)
+
+If renovate was never activated in an account on GitHub, the process starts by installing the [renovate app][gh-marketplace].
+
+To add an existing project to renovate (after renovate was installed) go to *Organization Account* -> *Installed GitHub Apps* for organizations or 
+*Account Settings* -> *Applications* for personal accounts. Click *Configure* on the *Renovate* application and add all repositories that should be 
+enabled for renovate.
+
+When renovate is activated first on a project, it starts the onboarding process by creating a pull request
```
Comment (devlead) [inline input/blog/2021-04-13-cake-support-in-renovate.md:26]:
```suggestion
When Renovate is activated first on a project, it starts the onboarding process by creating a pull request
```

File: input/blog/2021-04-13-cake-support-in-renovate.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+<!--excerpt-->
+
+### Enabling Renovate (on GitHub)
+
+If renovate was never activated in an account on GitHub, the process starts by installing the [renovate app][gh-marketplace].
+
+To add an existing project to renovate (after renovate was installed) go to *Organization Account* -> *Installed GitHub Apps* for organizations or 
+*Account Settings* -> *Applications* for personal accounts. Click *Configure* on the *Renovate* application and add all repositories that should be 
+enabled for renovate.
```
Comment (devlead) [inline input/blog/2021-04-13-cake-support-in-renovate.md:24]:
```suggestion
enabled for Renovate.
```

File: input/blog/2021-04-13-cake-support-in-renovate.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: Cake support in WhiteSource Renovate
+category: News
+author: nils-a
+---
+
+[WhiteSource Renovate][renovate] is a solution for automating dependency updates in software projects which works (among others)
+hosted in GitHub (much like dependabot does).
+
+In the new [version 24.114.0][cake-in-renovate-release] renovate added support for Cake files!
```
Comment (devlead) [inline input/blog/2021-04-13-cake-support-in-renovate.md:10]:
```suggestion
In the new [version 24.114.0][cake-in-renovate-release] Renovate added support for Cake files! 
```

File: input/blog/2021-04-13-cake-support-in-renovate.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+So, as of now it is possible to have pull requests automatically created to update `#addin`, `#tool` and `#module` references in Cake files.
+
+Big thanks go to the people behind WhiteSource Renovate for getting this done so quickly!
+
+<!--excerpt-->
+
+### Enabling Renovate (on GitHub)
+
+If renovate was never activated in an account on GitHub, the process starts by installing the [renovate app][gh-marketplace].
```
Comment (devlead) [inline input/blog/2021-04-13-cake-support-in-renovate.md:20]:
```suggestion
If Renovate was never activated in an account on GitHub, the process starts by installing the [Renovate app][gh-marketplace].
```

**Review by devlead — COMMENTED** [review]
LGTM 👍 Some minor suggestions.

File: input/blog/2021-04-13-cake-support-in-renovate.md (hunk 4 above)
Comment (nils-a) [reply input/blog/2021-04-13-cake-support-in-renovate.md]: 👍

File: input/blog/2021-04-13-cake-support-in-renovate.md (hunk 2 above)
Comment (nils-a) [reply input/blog/2021-04-13-cake-support-in-renovate.md]: 👍

File: input/blog/2021-04-13-cake-support-in-renovate.md (hunk 1 above)
Comment (nils-a) [reply input/blog/2021-04-13-cake-support-in-renovate.md]: 👍

**Pull request merged**